docs/getting-started.md

```diff
@@ -8,8 +8,11 @@ You can check if your system meets the requirements by running
 An opinionated way to run Firecracker is to launch an
 [EC2](https://aws.amazon.com/ec2/) `c5.metal` instance with Ubuntu 22.04.
 
-EC2 only supports nested virtualization on metal instances, which is why we use
-`.metal` instances exclusively.
+Firecracker requires [the KVM Linux kernel module](https://www.linux-kvm.org/)
+to perform some of its virtualization and emulation tasks.
```

Comment:
I think we can be more specific here.

Reply:
which details would you like to include?

Reply:
removed "some", thanks.